python/tests/test_tables.py

```diff
@@ -851,65 +851,75 @@ def test_round_trip_set_columns(self):
                 assert table[j].metadata == metadata_column[j]
                 assert table[j + num_rows].metadata == metadata_column[j]
 
-    def test_set_null_metadata(self):
-        for codec in ["struct", "json"]:
-            table = self.table_class()
-            table.metadata_schema = metadata.MetadataSchema(
-                {
-                    "codec": f"{codec}",
-                    "title": "Example Metadata",
-                    "type": ["object", "null"],
-                    "properties": {
-                        "a": {"type": "number", "binaryFormat": "i"},
-                    },
-                    "required": ["a"],
-                    "additionalProperties": False,
+    # Properties are not optional for struct
```

Comment:
Is this because the default for struct is to require properties, and not for json?

Reply:
Yes, struct has no way to encode missingness in the current design.

Reply:
Ah, never mind, I got it.